pkg/apiclient/apiclient.go

```diff
@@ -349,14 +349,14 @@ func (c *client) refreshAuthToken(localCfg *localconfig.LocalConfig, ctxName, co
 		return err
 	}
 	parser := &jwt.Parser{
-		ValidationHelper: jwt.NewValidationHelper(jwt.WithoutClaimsValidation()),
+		ValidationHelper: jwt.NewValidationHelper(jwt.WithoutClaimsValidation(), jwt.WithoutAudienceValidation()),
```

Comment:
@alexmt Should we disable audience validation for all jwt-go/v4 parsers? 
Based on https://github.com/argoproj/argo-cd/pull/5184/files

Reply:
I think you are right. Nice catch! We need to update all other places as well.